lib/Word_Lib/Enumeration_Word.thy

```diff
@@ -2,18 +2,15 @@
  * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
  *
  * SPDX-License-Identifier: BSD-2-Clause
+Proofs tidied by LCP, 2024-09
```

Comment:
I thought I had moved these to the AFP changelog. Looks like I still need to apply that patch.

Reply:
now applied.